### PR TITLE
create symptoms css, js  file 

### PR DESCRIPTION
**Pull request by NadaSaleh20**
- Making Style to the header
- Making Style to section 
- Making the Style responsive in CSS file 
- Making Dark mode code in the js file, move to another mode when clicking on the dark icon
- Making Style to color in the dark mode 
- Making function in js that move to next page when clicking on next  button 
- Making function in js to move to the previous page when clicking on the back icon 
 - Relates #28 
 - Relates #31

### CODE REVIEW

**Review by HebaAhmad1 — APPROVED** [review]

File: symptoms/symptoms.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//dark mode 
+var icon = document.getElementById("icon");
```
Comment (Ibrahim-Jarada) [inline symptoms/symptoms.js:2]: Try to avoid using var as you can, its preferable to use const here instead of var

File: symptoms/symptoms.html [hunk 2]
```diff
@@ -40,7 +40,7 @@ <h1 class="title">Breathing Difficulty</h1>
         <p class="para">Contact your doctor or clinic right away if you have COVID-19 symptoms, you’ve been exposed to someone with COVID-19.</p>
     </section>
   </div>
-   <div class="button"><button class="btn next">Next</button> </div> 
+  <div id="button"><button class="btn next">Next</button> </div>
```
Comment (Ibrahim-Jarada) [inline symptoms/symptoms.html:43]: recommend to use meaningful and related id, I suggest to use "next-btn"

File: symptoms/symptoms.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+icon.onclick = function(){
+    document.body.classList.toggle("dark-mode");
+    if ( document.body.classList.contains("dark-mode")) {
+        icon.src="../assets/sun.svg";
+
+    }else{
+        icon.src="../assets/dark-mode.png";
+    }
+}
+document.getElementById("button").onclick = function () {
```
Comment (Ibrahim-Jarada) [inline symptoms/symptoms.js:12]: Why didn't you assign your element to a const with meaningful name like icon?
if you may use your element in your code in future just assign it to const at the top of the code

File: symptoms/symptoms.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    location.href = "../advice/advice.html";
+
+}
+document.getElementById("back").onclick = function () {
```
Comment (Ibrahim-Jarada) [inline symptoms/symptoms.js:16]: Same as the above comment

**Review by AbdallahGot — CHANGES_REQUESTED** [review]
image
why its look like that !!!
try to avoid the horizontal scroll here.
to solve this you should use the percentage instead of using pixel in your style.
for example when we use this size of the phone its look like that 👇👇
image
so using the percentage will help you, especially in responsive design.

**Review by Rawanzaqout — APPROVED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by HebaAhmad1 — APPROVED** [review]

**Review by MontherIsmail — APPROVED** [review]

**Review by lina-jamal — APPROVED** [review]

**Pull request merged**